templates/partners/iot-device.html

```diff
@@ -0,0 +1,99 @@
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-12">
+      <ul class="p-inline-images">
+        {% for partner in partners %}
+        <li class="p-inline-images__item">
+          <img class="p-inline-images__logo" src="{{ partner.logo }}" style="min-width: 5rem;" alt="{{ partner.slug}}">
+        </li>
+        {% endfor %}
+      </ul>
+    </div>
+    <div class="row">
+      <div class="col-12 u-align--center">
+        <p style="max-width: none;"><a href="/partners/find-a-partner">See all partners&nbsp;&rsaquo;</a></p>
+      </div>
+    </div>
+  </div>
+</section>
+<section class="p-strip--light is-deep">
+  <div class="u-fixed-width">
+    <h2>Benefits of the IoT Partner Programme</h2>
+  </div>
+  <div class="row">
+    <div class="col-4">
+      <h3>Ubuntu Certified Hardware</h3>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">All Ubuntu functionality works out-of-the-box </li>
+        <li class="p-list__item is-ticked">Assistance to test, review, and configure machines to be Ubuntu Certified</li>
+        <li class="p-list__item is-ticked">Reduce time to market by deployment-ready hardware</li>
+      </ul>
+    </div>
+    <div class="col-4">
+      <h3>Long term support </h3>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Dedicated partner team and sales enablement access</li>
+        <li class="p-list__item is-ticked">Access technical guides to enhance team’s knowledge</li>
+        <li class="p-list__item is-ticked">Support and security updates for up to 10 years</li>
+      </ul>
+    </div>
+    <div class="col-4">
+      <h3>Joint marketing</h3>
+      <p>Logos and joint branding</p>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Logos and joint branding</li>
+        <li class="p-list__item is-ticked">Public postings on Ubuntu page</li>
+        <li class="p-list__item is-ticked">Promote your hardware as Ubuntu Certified</li>
+      </ul>
+    </div>
+  </div>
+</section>
+<section class="p-strip is-deep">
+  <div class="u-fixed-width">
+    <h2>Accelerate your business with Ubuntu Certified Hardware</h2>
+  </div>
+  <div class="row">
+    <div class="col-6">
+      <p>Our users have a high level of expectation in their experience with Ubuntu. Whether at home, in development, or production, an easy and predictable user experience is important to us. To ensure this success, we partner closely with vendors worldwide to test, validate, and certify their portfolios to ensure this success.</p>
+      <p>Our partners gain access to a worldwide audience looking to purchase devices built to run on Ubuntu by participating in the partner program. The top manufacturers in the world are all engaged with Canonical on promoting their platforms as Ubuntu Certified.</p>
+      <p><a href="https://ubuntu.com/certified" class="p-link--external">Learn more about Ubuntu certified hardware</a></p>
```

Comment:
Ubuntu Certified Hardware is capitalised everywhere. Can you check which one it is please.